backend/session/models.py

```diff
@@ -75,15 +80,25 @@ def load_json_data(self):
         """Get json data as object"""
         return self.json_data if self.json_data else {}
 
+    def save_json_data_key(self, key: str, value):
```

Comment:
What are these two methods used for?

Reply:
I was using them, but then I think I didn't need them anymore so I'll remove the methods as well here.